autoscale_cloudroast/test_repo/autoscale/fixtures.py

```diff
@@ -72,16 +71,17 @@ def setUpClass(cls):
 
         cls.tenant_id = cls.autoscale_config.tenant_id
 
-        env = os.environ['OSTNG_CONFIG_FILE']
-        if ('preprod' in env.lower()) or ('dev' in env.lower()):
-            cls.url = str(cls.autoscale_config.server_endpoint) + \
-                '/' + str(cls.tenant_id)
-            print(" ------ Using dev or pre-prod otter --------")
-        else:
+        # If not production or staging, always use the configured server
```

Comment:
Minor nitpick: Do you think moving this comment to below `else:` statement will be easier to read?